struct_amd64.go

```diff
@@ -6,8 +6,47 @@ package purego
 import (
 	"math"
 	"reflect"
+	"unsafe"
 )
 
+func getStruct(outType reflect.Type, syscall syscall15Args) (v reflect.Value) {
+	outSize := outType.Size()
+	if outSize == 0 {
+		return reflect.New(outType).Elem()
+	} else if outSize <= 8 {
```

Comment:
We don't need `else`, right?

Reply:
I mean, why not

```go
	if outSize == 0 {
		return reflect.New(outType).Elem()
	}
	if outSize <= 8 {
```

Reply:
Are you saying just for the first `else if`? If so, idc either way but I was thinking grouping them made it clear that the conditionals build off of each other. ==0, <=8, <=16, >16

Reply:
All the `else if`. If you want to make it explicit that all the conditions are exclusive, I prefer switch-case.

```go
switch {
case outSize == 0:
    return reflect.New(outType).Elem()
case outSize <= 8:
```